src/transform.ts

```diff
@@ -168,27 +169,15 @@ export async function compileFile(
   }
 
   // styles
-  const ceFilter = store.customElement
```

Comment:
Removed `customElement` option, now get it from `sfcOptions.script.customElement`.